Flutter/cartesian-charts/methods.md

```diff
@@ -653,6 +813,336 @@ The [`hide`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/char
+          ..isAntiAlias = true
+          ..strokeCap = StrokeCap.round
+          ..color = Colors.red
+          ..strokeWidth = 2;
+        dashArray = [10, 15, 10];
+        super.drawHorizontalAxisLine(context, offset, dashArray, strokePaint);
+      }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### DrawHorizontalAxisTooltip method in crosshairBehavior
+
+To customize the tooltip label, position, and style for horizontal axis tooltip.
+
+* `context` - Specifies the painting context.
+* `position` - Specifies the position of the tooltip label.
+* `text` - Specifies the tooltip label.
+* `style` - Specifies the style of the tooltip text.
+* `path` - Specifies path reference of the tooltip rectangle.
+* `fillPaint` - Specifies the fill paint applied to the default tooltip rectangle.
+* `strokePaint` - Specifies the stroke paint applied to the default tooltip rectangle.
+
+{% tabs %}
+{% highlight dart %} 
+
+    class CustomCrosshair extends CrosshairBehavior {
+      Offset? customPosition;
+
+      @override
+      void show(x, double y, [String coordinateUnit = 'point']) {
+        if (coordinateUnit == 'pixel') {
+          customPosition = Offset(x.toDouble(), y);
+        }
+        super.show(x, y, 'pixel');
+      }
+
+      @override
+      void drawHorizontalAxisTooltip(PaintingContext context, Offset position, String text, TextStyle style, [Path? path, Paint? fillPaint, Paint? strokePaint]) {
+        TextStyle style = TextStyle(
+        color: Colors.white,
+        fontSize: 12,
+        background: Paint()
+          ..color = Colors.blueGrey
+          ..strokeWidth = 16
+          ..strokeJoin = StrokeJoin.round
+          ..style = dart_ui.PaintingStyle.stroke,
+        );
+
+        if (parentBox != null) {
+        final Rect bounds = parentBox!.paintBounds;
+        final String finalText = 'Y : ${verticalValue.toStringAsFixed(2)}';
+        final Offset horizontalPosition =
+            Offset(bounds.right, customPosition!.dy).translate(-50, -20);
+        _drawText(context.canvas, finalText, horizontalPosition, style);
+      }
+    }
+
+      void _drawText(Canvas canvas, String text, Offset point, TextStyle style) {
+        final TextPainter textPainter = TextPainter(
+          text: TextSpan(
+            text: text,
+            style: style.copyWith(fontWeight: dart_ui.FontWeight.bold)),
+        textDirection: dart_ui.TextDirection.ltr,
+        textAlign: TextAlign.center,
+        maxLines: getMaxLinesContent(text),
+        );
+        
+        textPainter
+        ..layout()
+        ..paint(canvas, point);
+      }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Crosshair Horizonatl Axis Tooltip](images\trackball-crosshair/custom_crosshair_horizontal_tooltip.png)
+
+### DrawVerticalAxisTooltip method in crosshairBehavior
+
+To customize the tooltip label, position, and style for vertical axis tooltip.
+
+For further reference, please consult the drawHorizontalAxisTooltip code snippet.
```

Comment:
For further reference, please consult the drawHorizontalAxisTooltip code sample.